.github/workflows/ci.yml

```
@@ -75,6 +75,33 @@ jobs:
       docker-file: ./HadesScheduler/HadesOperator/Dockerfile
     secrets: inherit
 
+  build-operator:
```

actionlint (1.7.11)

[error] key "build-operator" is duplicated in "jobs" section. previously defined at line:69,col:3. note that this key is case insensitive

(syntax-check)